components/style/properties/longhand/list.mako.rs

```diff
@@ -30,69 +30,13 @@ ${helpers.single_keyword("list-style-position", "outside inside", animation_valu
         animation_value_type="discrete",
         spec="https://drafts.csswg.org/css-lists/#propdef-list-style-type")}
 % else:
```

Comment:
Should I try to remove the conditional here? If so, please could I have some pointers? (The two branches seem quite different and therefore tricky to know how to homogenise them).

Reply:
Yeah, I agree it's non-trivial to do so, so let's keep this for now.